build/sbt-launch-lib.bash

```diff
@@ -186,7 +186,7 @@ run() {
     ${SBT_OPTS:-$default_sbt_opts} \
     $(get_mem_opts $sbt_mem) \
     ${java_opts} \
-    ${java_args[@]} \
+    "${java_args[@]}" \
```

Comment:
SC2068 (error): Double quote array expansions to avoid re-splitting elements.

Reply:
I think this is ok, my one concern would be someone _might_ be depending on the shell expansion here for something like class path but since this is a build only tool I think that's an OK risk to take given the tradeoffs.